src/components/Accordion/Accordion.stories.tsx

```diff
@@ -33,7 +33,7 @@ const AccordionExample = () => {
       </Accordion>
       <br />
       <p>Accordion - stay open until closed</p>
-      <Accordion>
+      <Accordion className="cc-accordion--0">
```

Comment:
Please check the explanation in this PR: https://github.com/wellcometrust/corporate-react/pull/436